### PR TITLE
Progress counter and sit-out Button.

### LINKED ISSUE #111: Skip button (Vertagen, Aussitzen)
We decided to allow the player to do nothing / skip a decision / pass no law. A button "Aussitzen" would trigger a next step in the game without passing a law. The `NewYear.ts` law also has to be modified, since it counts passed laws.

### PR DESCRIPTION
**Pull request by mdrie**
Hey Joachim,

before this change it took far too long to finish a year. This was, because it was done by counting the laws accepted in a year. However, events and other laws may un-accept already accepted laws. Therefore, we need the independent counter. 

With this change, the "Aussitzen" button was simple - at least this ugly one.

If we decide not to have it in the game, we can always hide it - as we did with the "reject" buttons of the laws.

closes #111

### CODE REVIEW

File: src/components/LawProposals.vue [hunk 1]
```diff
@@ -80,4 +85,22 @@ export default defineComponent({
...
+  transition-duration: 0.1s;
+}
+
+.sitOutButton:hover {
+  background: lightgrey;
+}
+
+.sitOutButton:active {
+  transform: translate3d(250px, 350px, 400px) scale(1.1);
+}
```
Comment (jschirrmacher) [inline src/components/LawProposals.vue:105]: How about showing the button only if the laws are lifted up?

**Comment by jschirrmacher** [comment]
We could hide the sit out button if the current law set does contain a "do nothing" law. This could be a boolean attribute in `LawDefinition` which can be searched for.

**Comment by mdrie** [comment]
Both are nice ideas. But do you think, this shouldn't be merged?

File: src/components/LawProposals.vue (hunk 1 above)
Comment (mdrie) [reply src/components/LawProposals.vue:105]: Nice idea. Right now, i (we?) just need the functionality to do more testing of the game. Details of the presentation are important - but in my opinion not now.

**Comment by mdrie** [comment]
I am not in favour of the "do nothing" laws - currently mainly because they are quite a bit of additional work. We hardly have any laws for industry and buildings and way too few "non-happy-path" laws in entergy and mobility. Not to speak of events...

**Comment by jschirrmacher** [comment]
Ok, but I will add another issue at least.

**Pull request merged**